src/components/SummaryCard/SummaryCard.stories.js

```diff
@@ -3,120 +3,182 @@
-                iconDescription="Icon description"
-                renderIcon={Folder20}
-                hasIconOnly
-                onClick={action('onClick')}
-                tooltipPosition="bottom"
-                tooltipAlignment="center"
-              />
-            </SummaryCardFooter>
-          </SummaryCard>
-        </div>
+          </SummaryCardFooter>
+        </SummaryCard>
       </div>
-    </div>
+      <div
+        className={`${prefix}--col-sm-4 ${prefix}--col-md-4 ${prefix}--col-lg-4`}
+      >
+        <SummaryCard>
+          <SummaryCardHeader
+            title="Summary card with no footer"
+            status={
+              <Tooltip showIcon iconDescription="Status">
+                Tooltip content
+              </Tooltip>
+            }
+          />
+          <SummaryCardBody>{lorem.repeat(5)}</SummaryCardBody>
+        </SummaryCard>
+      </div>
+      <div
+        className={`${prefix}--col-sm-4 ${prefix}--col-md-4 ${prefix}--col-lg-4`}
+      >
+        <SummaryCard>
+          <SummaryCardHeader title="Summary card with no status" />
+          <SummaryCardBody>{lorem.repeat(3)}</SummaryCardBody>
+          <SummaryCardFooter>
+            <SummaryCardAction loading expandedContent={lorem.repeat(5)}>
+              Button label
+            </SummaryCardAction>
+            <SummaryCardAction
+              iconDescription="Icon description"
+              renderIcon={Folder20}
+              hasIconOnly
+              onClick={action('onClick')}
+              tooltipPosition="bottom"
+              tooltipAlignment="center"
+            />
+            <SummaryCardAction
+              iconDescription="Icon description"
+              renderIcon={Folder20}
+              hasIconOnly
+              onClick={action('onClick')}
+              tooltipPosition="bottom"
+              tooltipAlignment="center"
+            />
+            <SummaryCardAction
+              iconDescription="Icon description"
+              renderIcon={Folder20}
+              hasIconOnly
+              onClick={action('onClick')}
+              tooltipPosition="bottom"
+              tooltipAlignment="center"
+            />
+          </SummaryCardFooter>
+        </SummaryCard>
+      </div>
+    </Fragment>
+  ))
+  .add('Multiselect', () => (
+    <SummaryCardContainer
+      render={({
+        getBatchActionProps,
+        getSelectionProps,
+        selectedSummaryCards,
+        summaryCards,
+      }) => (
+        <Fragment>
+          <SummaryCardBatchActions {...getBatchActionProps()}>
+            <SummaryCardBatchAction
+              onClick={() =>
+                action('SummaryCardBatchAction onClick')(selectedSummaryCards)
+              }
+              renderIcon={Delete16}
+            >
+              Action
+            </SummaryCardBatchAction>
+          </SummaryCardBatchActions>
+
+          <div className="bx--row">
+            {summaryCards.map(({ id }) => (
```

Comment:
Unfortunately this story structure makes the source look strange & does not reflect the structure of the component at all:

![Screen Shot 2019-12-05 at 1 33 29 PM](https://user-images.githubusercontent.com/9057921/70267357-e9353180-1763-11ea-93a7-9548f9b2ff6f.png)

I can see someone getting confused when they see that, and they wouldn't be able to copy + paste (which I always think of as a very ideal scenario, personally)